src/Drawer/index.js

```diff
@@ -153,13 +157,23 @@ const Drawer = ({
       style={depthStyle}
       className={`drawer drawer--${position} ${
         isOpen && isTransitioning ? `drawer--open--${position}` : ""
-      }`}
+      } ${isUsingMobileDrawer ? "drawer--mobile" : ""}`}
```

Comment:
`import cc from "classcat"` will help here - a lot of components in NDS already use it to compose conditional classnames

Reply:
```jsx
className={cc([
    "thisClassIsAlwaysApplied",
    "soIsThisOne",
    {
        "conditionallyAddedClass": true, // will be added
        "AnotherconditionallyAddedClass": false, // will NOT be added
    }
])}
```